ami/jobs/management/commands/test_ml_job_e2e.py

```
@@ -0,0 +1,127 @@
+    def handle(self, *args, **options):
+        collection_name = options["collection_name"]
+        pipeline_slug = options["pipeline_slug"]
+        dispatch_mode = options["dispatch_mode"]
+
+        # Find collection
+        try:
+            collection = SourceImageCollection.objects.get(name=collection_name)
+            self.stdout.write(self.style.SUCCESS(f"✓ Found collection: {collection.name}"))
+            self.stdout.write(f"  Project: {collection.project.name}")
+        except SourceImageCollection.DoesNotExist:
+            raise CommandError(f"SourceImageCollection '{collection_name}' not found")
```

Ruff (0.15.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)